src/styles/theme.ts

```diff
@@ -47,6 +47,7 @@ export const colors: Colors = {
   // Semantic colors
   primary: white,
   secondary: '#888F96',
+  border: '#2A2A2A',
```

Comment:
im going to suggest one thing (not blocking in any way right now), that we use CSS variables for our theme instead of styled components theme props. 
Here is a couple of reasons:
- CSS variables changing don't cause re-renders if changed
- They are a browser built in and reactive by default 
- They have no runtime cost